### PR TITLE
Added arguments() method on solver command object

### PR DESCRIPTION
**Pull request by hpohekar**
Now we are getting results similar to `iterate.get_attr("arguments")` with `iterate.arguments()`

image

### CODE REVIEW

File: src/ansys/fluent/core/solver/flobject.py [hunk 1]
```diff
@@ -150,6 +150,14 @@ def get_attr(self, attr) -> Any:
             raise RuntimeError("Object is not active")
         return attrs[attr] if attrs else None
 
+    def arguments(self) -> Any:
+        attrs = self.get_attrs(["arguments"])
+        if attrs:
+            attrs = attrs.get("attrs", attrs)
+        if "arguments" != "active?" and attrs and attrs.get("active?", True) is False:
```
Comment (seanpearsonuk) [inline src/ansys/fluent/core/solver/flobject.py:157]: `"arguments" != "active?"` part is redundant

Comment (hpohekar) [reply src/ansys/fluent/core/solver/flobject.py:157]: Ok. Thank you.

File: src/ansys/fluent/core/solver/flobject.py [hunk 2]
```diff
@@ -150,6 +150,14 @@ def get_attr(self, attr) -> Any:
...
+            raise RuntimeError("Object is not active")
```
Comment (seanpearsonuk) [inline src/ansys/fluent/core/solver/flobject.py:158]:
```suggestion
            raise RuntimeError("Command is not active")
```

Comment (hpohekar) [reply src/ansys/fluent/core/solver/flobject.py:158]: Ok. Thank you.

**Review by prmukherj — APPROVED** [review]

File: src/ansys/fluent/core/solver/flobject.py (hunk 1 above)
Comment (hpohekar) [reply src/ansys/fluent/core/solver/flobject.py]: Done.

File: src/ansys/fluent/core/solver/flobject.py (hunk 2 above)
Comment (hpohekar) [reply src/ansys/fluent/core/solver/flobject.py]: Done.

**Comment by seanpearsonuk** [comment]
@hpohekar I don't see any issues in the current version. I would just like to add @h-krishnan as he is the original author of this code.

**Comment by hpohekar** [comment]
> @hpohekar I don't see any issues in the current version. I would just like to add @h-krishnan as he is the original author of this code.

Yes sure. Thank you.

**Review by seanpearsonuk — APPROVED** [review]

**Review by mkundu1 — APPROVED** [review]

**Comment by mkundu1** [comment]
@hpohekar Do we have a Fluent side PR for syncing?

**Comment by hpohekar** [comment]
> @hpohekar Do we have a Fluent side PR for syncing?

Not yet.

**Comment by hpohekar** [comment]
> > @hpohekar I don't see any issues in the current version. I would just like to add @h-krishnan as he is the original author of this code.
> 
> Yes sure. Thank you.

Hi @h-krishnan , Please share your views. Thank you.

**Pull request merged**